Sources/PointFree/Episode/Show.swift

```diff
@@ -13,15 +13,18 @@ import Styleguide
 import Tuple
 import Views
 
-let episodeResponse =
-  fetchEpisodeForParam
-    <| writeStatus(.ok)
+let episodeResponse
+  : M<Tuple4<Episode, User?, SubscriberState, Route>>
+  = writeStatus(.ok)
     >=> userEpisodePermission
     >=> map(lower)
     >>> respond(
       view: Views.episodeView(episodePageData:),
       layoutData: { permission, episode, currentUser, subscriberState, currentRoute in
         let navStyle: NavStyle = currentUser == nil ? .mountains(.main) : .minimal(.light)
+        #if DEBUG
+        var episode = Current.episodes().first(where: { $0.sequence == episode.sequence }) ?? episode
+        #endif
```

Comment:
These lines are necessary for routing to go through tests. You might have a better idea...

Reply:
one thought comes to mind, we could inject the episodes into the router here: https://github.com/pointfreeco/pointfreeco/blob/master/Sources/PointFree/Bootstrap.swift#L88